### PR TITLE
Redesign by jeppes ✌️

### PR DESCRIPTION
**Pull request by jeppes**
| Before | After |
|--------|--------|
| Screenshot 2023-12-05 at 20 46 30 | Screenshot 2023-12-05 at 20 45 29 |

The emoji changes when refreshing the page from a fixed list of emoji

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,19 +18,55 @@
...
+      ];
+      var emoji = emojis[index % emojis.length];
+      // Append a random emoji to the #title element
+      const titleElement = document.getElementById("title");
+      titleElement.textContent += emoji;
+
+      localStorage.setItem("index", (index + 1) + "");
+    </script>
+
+    <script>
```
Comment (younjin) [inline index.html:47]: Does this need to be here? 😓

**Comment by younjin** [comment]
Looks great!

How does the resume look locally (with the code changes)?

**Comment by younjin** [comment]
@jeppes: How does #4A96E8 look? With either #4AC7E8 or #14539A as the accent colour?

**Comment by jeppes** [comment]
Screenshot 2023-12-05 at 21 34 46

Updated design!

**Comment by younjin** [comment]
:shipit:

**Pull request merged**